.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -18,6 +18,17 @@ Tell us what should happen
 ### Actual behaviour
 Tell us what happens instead
 
+## Talk app
+
+**Talk app version:** (see apps admin page: `/index.php/settings/apps`)
+
+**Custom Signaling server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
+
+**Custom TURN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
+
+**Custom STUN server configured:** yes/no (see additional admin settings: `/index.php/settings/admin/additional`)
```

Comment:
Shouldn't this be `/index.php/settings/admin/talk#stun_server` or, if that is too specific, `/index.php/settings/admin/talk`?